src/Orc.Controls/Orc.Controls.Shared/Controls/DropDownButton/Views/DropDownButton.xaml.cs

```diff
@@ -13,8 +13,32 @@ namespace Orc.Controls
     using System.Windows.Input;
     using System.Windows.Media;
     using Catel;
-    using Catel.MVVM.Views;
-    using Microsoft.WindowsAPICodePack.Shell;
+
+    /// <summary>
+    /// The arrow location.
+    /// </summary>
+    public enum ArrowLocation
```

Comment:
Maybe name it "DropDownArrowLocation"? We might need this in other controls as well in the future.